HTAN.model.csv

```diff
@@ -53,6 +53,7 @@ Bulk WES Level 2,Bulk Whole Exome Sequencing aligned files and QC,,"Component, F
 Bulk WES Level 3,Bulk Whole Exome Sequencing called variants,,"Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Genomic Reference, Genomic Reference URL, Germline Variants Workflow URL, Germline Variants Workflow Type, Germline Variants Workflow Type, Somatic Variants Workflow URL, Somatic Variants Workflow Type, Somatic Variants Sample Type, Structural Variant Workflow URL, Structural Variant Workflow Type",,FALSE,Sequencing,Bulk WES Level 2,,
 scATAC-seq Level 1,"scATAC-seq files containing sequence read information, with or without alignment, as FASTQ or BAM files",,"Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Nucleic Acid Source, Dissociation Method, Single Nucleus Buffer, Single Cell Isolation Method, Transposition Reaction, scATACseq Library Layout, Nucleus Identifier, Nuclei Barcode Length, Nuclei Barcode Read, scATACseq Read1, scATACseq Read2, scATACseq Read3, Library Construction Method, Sequencing Platform, Threshold for Minimum Passing Reads, Total Number of Passing Nuclei, Median Fraction of Reads in Peaks, Median Fraction of Reads in Annotated cis DNA Elements, Median Passing Read Percentage, Median Percentage of Mitochondrial Reads per Nucleus,Technical Replicate Group, Total Reads, Protocol Link",,FALSE,Sequencing,Biospecimen,,
 scATAC-seq Level 4,"Data represents the relationships between cells derived from Level 3 expression data and shown as tSNE or UMAP coordinates per cell, plus all other cell-specific meta information (e.g., cell type)",,"Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, scATACseq Workflow Type, scATACseq Workflow Parameters Description, Workflow Version, Workflow Link, Workflow Start Datetime, Workflow End Datetime",,FALSE,Sequencing,scATAC-seq Level 3,,
+Imaging Level 1,Raw imaging data,,"Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Imaging Assay Type, Protocol Link, Software and Version, Commit SHA, Pre-processing Completed, Pre-processing Required, Comment",,FALSE,Assay,Biospecimen,,

```

Comment:
Attributes `Imaging Assay Type`, `Protocol Link`, `Software and Version`, and `Commit SHA` have `Imaging Level 2` as their parent. is it OK to use them here? If not do they need to be duplicated with `Imaging Level 3` as there parent or split out into a more general `Imaging` parent as done for `Sequencing?